engine/runtime/src/test/scala/org/enso/compiler/test/CompilerTest.scala

```diff
@@ -61,7 +64,13 @@ trait CompilerRunner {
       * @return the [[IR]] representing [[source]]
       */
     def toIrModule: IR.Module = {
-      AstToIr.translate(source.toAst)
+      if (useRustParser) {
+        val compiler = new EnsoCompiler()
+        try compiler.compile(source)
+        finally compiler.close()
+      } else {
+        AstToIr.translate(source.toAst)
```

Comment:
Is there any need to test the old `AstToIr`? The Rust based parser shall deliver good enough results. If not, such a test shall be added to `EnsoCompilerTest` - that would show the difference and the difference could be eliminated.

Reply:
New parser produces a lot of errors with the compiler test snippets. It will take significant time to go through each case and fix it.

Reply:
This will be problematic soon as we want to ditch it quickly. Please file a separate ticket or (if it exists already) make sure this one is included in the tasks.